dist/tools/jlink/jlink.sh

```diff
@@ -58,7 +58,12 @@ _JLINK_SERVER=JLinkGDBServer
 _JLINK_IF=SWD
 _JLINK_SPEED=2000
 # default starting address of the devices flash memory
-_FLASH_ADDR=0
+if [ -n "${FLASH_OFFSET}" ]; then
+    _FLASH_ADDR=${FLASH_OFFSET}
+    echo "Flashing at offset ${FLASH_OFFSET}"
+else
+    _FLASH_ADDR=0
+fi
```

Comment:
> So I think due to these differences a "super global" approach seems to me complicated to implement without big changes on all the scripts

I disagree: having this 'standard' global variable does not hurt in any case. If the flash tool does not need it, simply don't use it. If there is something (whatever that my be) that needs to know the starting address of the flash, we have a defined way to define/retrieve it.